src/App.tsx

```diff
@@ -73,6 +81,12 @@ const App = () => {
     dispatch(fetchData())
   }, [isRestored, isBackedUp, dispatch])
 
+  useEffect(() => {
+    if (isBackedUp) {
```

Comment:
Whats the flow on this? It seems like something that should happen once on launch of the app? 

Why do we do it only if `isBackedUp` is true? And we need to run again if it goes false and back to true. Im wondering if just passing [] would work?

Reply:
We need to wait for it to be backed up so that the account address is defined. `client.account(address).activity.list()` requires account, so we reinitialize them whenever `isBacked` changes to true.

Reply:
Ah ok, that makes sense